app/api/items/get_breadcrumbs_from_roots.feature

```diff
@@ -131,3 +138,7 @@ Feature: Find all breadcrumbs to an item
         [{"id": "60", "title": "Reduce Graph", "language_tag": "en", "type": "Task"}]
       ]
       """
+    Examples:
+      | service_url                                   |
+      | /items/60/breadcrumbs-from-roots              |
+      | /items/by-text-id/id60/breadcrumbs-from-roots |
```

Comment:
Probably it is also worth testing that url decoding work as expected (encoding of spaces, slashes, ...) as `text_id` is a string

Reply:
fixed. It only allows alphanum characters now to avoid issues. MySQL migration added to remove other characters. Doc updated too.

The migration strips all non-alphanum characters including accented characters. "créée" will be updated as "cre".

Reply:
To be sure: So you require the text_id to no non-alphanum char? 
 (so sure that's ok, as it is an external id)

Reply:
Yes, only a-z, A-Z and 0-9. We could add more but if possible it would be better if we whitelist explicitly the characters, as many characters can potentially make the url parsing fail, like #, ?, /, and we'll most likely forget to test them all.

Reply:
If you do classic url encoding/decoding, you'll most characters we need to support without having to care about #, ?, /, ...

Reply:
fixed. using url-encoding/decoding